members/C001113.yaml

```diff
@@ -74,7 +79,61 @@ contact_form:
           value: $TOPIC
           required: true
           options:
+            Abortion: Abortion
+            Adoption: Adoption
+            Aging: Aging
+            Agriculture: Agriculture
+            Animal Welfare: Animal Welfare
             Appropriations: Appropriations
+            Arts and Humanities: Arts and Humanities
+            Banking: Banking
+            Campaign Finance: Campaign Finance
+            Civil Rights: Civil Rights
+            Commerce: Commerce
+            Communications: Communications
+            Congratulations: Congratulations
+            Crime: Crime
+            Defense: Defense
+            Drugs: Drugs
+            Economy: Economy
+            Education Higher: Education Higher
+            Elections: Elections
+            Energy: Energy
+            Environment: Environment
+            Family: Family
+            Food Stamps: Food Stamps
+            Foreign Affairs: Foreign Affairs
+            Gun Violence Prevention: Gun Violence Prevention
+            Health: Health
+            Homeland Security: Homeland Security
+            Housing: Housing
+            Immigration: Immigration
+            Insurance Health: Insurance Health
+            Insurance Other: Insurance Other
+            Interior: Interior
+            Judiciary: Judiciary
+            Labor: Labor
+            LGBTQ: LGBTQ
+            Medicaid: Medicaid
+            Medicare: Medicare
+            Military: Military
+            NASA: NASA
+            National Security: National Security
+            Native Americans: Native Americans
+            Nutrition: Nutrition
+            Patents: Patents
+            Pensions: Pensions
+            Public Lands: Public Lands
+            Securities: Securities
+            Small Business: Small Business
+            Social Security: Social Security
+            Taxes: Taxes
+            Trade: Trade
+            Transportation: Transportation
+            Veterans: Veterans
+            Water: Water
+            Welfare: Welfare
+            "Women's Issues": "Women's Issues"
```

Comment:
does it matter this one's in quotes?

Reply:
It doesn't. But for simplicity I took the quotes out